android/app/src/main/java/org/fossasia/openevent/api/DataDownloadManager.java

```diff
@@ -16,7 +20,8 @@
 public final class DataDownloadManager {
     private static DataDownloadManager instance;
 
-    private APIClient client = new APIClient();
+    private APIClient openEventClient = new APIClient(true);
```

Comment:
The boolean parameter is not clear for this purpose. APIClient true what?

Rather, move the creation of Retrofit service in their respective methods like getOpenEventApi and getFacebookApi, not in construcor